src/main/docs/5 - Flow Control/5.1 - If Else/description.md

```diff
@@ -54,6 +54,35 @@ You can have as many of these if/else chains as necessary.
 
 Imagine you had a third option in addition to “buy” and “hold”, “sell”. You could chain if/else expressions together to account for this additional criteria:
 
+## Conditional If
+
+Additionally, the **if** operator allows to conditionally add/remove a key/pair element into an object. 
+
+#### Input:
+```json
+{
+    "welcome-text": "Hello world!"
+}
+```
+#### DW Script:
+```dw
+%dw 2.0
+output application/json
+---
+{
+    (message: payload.message ) if (payload.message?)
```

Comment:
I would say this needs to go to Object part. We call this conditional key value pairs.